agent/util-scripts/pbench-tool-meister-start

```diff
@@ -850,35 +854,36 @@ def main(_prog, cli_params):
         ssh_opts=ssh_opts,
     )
 
-    orchestrate = cli_params.orchestrate
-    if orchestrate == "create":
+    _orchestrate = cli_params.orchestrate
```

Comment:
Not a big deal, but why bother instantiating `_orchestrate` when what we care about is the value of `orchestrate`?

I would have been tempted to create a helper function which accepts `cli_params` and returns the value for `orchestrate` -- it could do all the asserting and testing that it wants, and this code would just work with the result.  (And, it would encapsulate the details of the command line parameter far away....)